Vault-Deployment/manifests/network-policies/networkpolicy-loki-ingress.yaml

```diff
@@ -0,0 +1,30 @@
+# https://kubernetes.io/docs/concepts/services-networking/network-policies
+# https://support.hashicorp.com/hc/en-us/articles/13444259109523-Information-in-regards-to-Audit-Log-when-running-Vault-in-Kubernetes
+#
+# Network policiy for controlling communication between Vault, OTLP/Promtail and Loki
+---
+# Allow ingress from OTLP/Promtail/Grafana
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: loki-allow-from-otlp
+  namespace: loki
+spec:
+  # An empty podSelector selects all pods in the namespace
+  podSelector: {}
+  policyTypes:
+    - Ingress
+  ingress:
+    - from:
+        - namespaceSelector:
```

Comment:
And once again, a lot of time got wasted until I finally grasped that I needed to repeat the `namespaceSelector` for each namespace to make it logical AND the conditions :smile: Also, `name` is not guaranteed as label on the namespaces, but `kubernetes.io/metadata.name` pretty much is!